gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -202,6 +205,23 @@ def run(self) -> None:
         process_graph_generator.save_graph()
 
         pairs_graph = self.retriever.create_computation_graph(self.loader)
+
+        image_vis_data = {}
+        for i in range(len(self.loader)):
```

Comment:
This should be optional, controlled by a flag. I think we also duplicate metrics and outputs in `REACT_METRICS_PATH`. Maybe we can use the same flag to decide whether we want to do this. It is non-trivial time for large datasets.

Reply:
addressed in a new commit